src/Venue/Bookings/ServiceFee.cs

```diff
@@ -0,0 +1,95 @@
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Ivvy.API.Venue.Bookings
+{
+    /// <summary>
+    /// A description of service fee applied to a booking item.
+    /// </summary>
+    public class ServiceFee
+    {
+        [JsonProperty("id")]
+        public int Id
+        {
+            get; set;
+        }
+
+        [JsonProperty("refType")]
+        public int RefType
```

Comment:
RefType is an enum:
1 = Room
2 = Room Extra
3 = Package
4 = Session
5 = Menu
6 = Beverage Package
7 = Product
8 = Resource
9 = Additional Item
10 = Service Fee